Sources/Core/Other/PasteboardMonitor.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  PasteboardMonitor.swift
+//  XestiMonitors
+//
+//  Created by Paul nyondo on 2018-03-15.
+//
+//  © 2016 J. G. Pusey (see LICENSE.md)
+//
+
+#if os(iOS)
+    import  UIKit
+    ///
+    /// A `PasteboardMonitor` instance monitors a pasteboard for changes to
+    /// its state.
+    ///
+    public class PasteboardMonitor: BaseNotificationMonitor {
+        
+        ///
+        /// Encapsulates changes to the contents of the pasteboard.
+        ///
+        
+        public enum Event {
+            ///
+            ///  Contents of the pasteboard object have changed.
+            ///
+            case didChange(Info)
+            ///
+            ///  Contents of the pasteboard object have been removed.
+            ///
+            case didRemove(Info)
+            
+        }
+        
+        public struct Info {
+            ///
+            ///  The added representation types stored as an array
+            ///  in the notification’s userInfo dictionary
+            ///
+            public let typesAdded: String
```

Comment:
If I'm reading the docs correctly, this should be an _array_ of strings. (Same for `typesRemoved`.)